docs/source/mac-install.rst

```diff
@@ -207,13 +232,15 @@ For Option 1:
 
 .. code-block:: console 
 
-    source /opt/homebrew/opt/lmod/init/profile
+   export BASH_ENV="/opt/homebrew/opt/lmod/init/profile"
+   source $BASH_ENV
 
 For Option 2:
 
 .. code-block:: console 
 
-    source /usr/local/opt/lmod/init/profile
+   export BASH_ENV="/usr/local/opt/lmod/init/profile"
+   source $BASH_ENV
```

Comment:
What's the point of making it a Bash variable before sourcing?

Reply:
I think @natalie-perlin was just switching it to a more common (albeit less concise) way of doing this, but I'll let her weigh in. I presume it would also allow users to use this variable again later, if desired, although I don't know under what circumstances they would need to.

Reply:
This syntax is similar to the standard Lmod initialization practice for the shell (bash, as suggested to be used by the supporting documentation).

Reply:
BASH_ENV is also a standard environmental variable used in bash shells